### ISSUE TITLE
memory usage 

### ISSUE THREAD

**Issue by lfz** [Project-MONAI/VISTA#20]
https://github.com/Project-MONAI/VISTA/blob/1e7f3cc3ec10c5e9b3bc89b14fdeb2949e45a8e2/monailabel/monaivista/lib/model/vista_point_2pt5/inferer.py#L94

this line use too much memory when image is big

![image](https://github.com/Project-MONAI/VISTA/assets/8953923/51f1a0d2-ff2f-4004-8224-2e28ed44c9c6)

![image](https://github.com/Project-MONAI/VISTA/assets/8953923/f33db5e9-8a2d-4dcc-8685-159582cd57b1)

the definition of variable "pred_volume“ varies in different functions. it is 104 channel prob logits at beginning, but changed to 1 channel segmentation finally. To reduce memory use, it should be claimed as 1 channel segmentation map
